geode-core/src/main/java/org/apache/geode/cache/wan/GatewaySender.java

```diff
@@ -400,4 +400,19 @@
 
   public int getMaxParallelismForReplicatedRegion();
 
+
+  /**
+   * Destroys the GatewaySender. Before destroying the sender, caller needs to to ensure that the
+   * sender is stopped so that all the resources (threads, connection pool etc.) will be released
+   * properly. Stopping the sender is not handled in the destroy. Destroy is carried out in
+   * following steps: 1. Take the lifeCycleLock. 2. If the sender is attached to any application
+   * region, throw an exception. 3. Close the GatewaySenderAdvisor. 4. Remove the sender from the
+   * cache. 5. Destroy the region underlying the GatewaySender.
+   * <p>
+   * In case of ParallelGatewaySender, the destroy operation does distributed destroy of the QPR. In
+   * case of SerialGatewaySender, the queue region is destroyed locally.
```

Comment:
Is this still accurate? I see in the code you now call stop before calling destroy, so does the sender still need to be stopped?

This looks like it's the internal documentation about what happens. Maybe right something more oriented towards the user for the public method?

Also add an @since Geode 1.1 tag.